File: lib-model/src/main/java/com/minesec/msa/client/sdk/payment/model/DownloadParamDto.java
package com.minesec.msa.client.sdk.payment.model;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;

/**
 * @author eric.song
 * @since 2022/8/26 12:50
 */
@Getter
public class DownloadParamDto implements Serializable {
    /**
     * global terminal EMV parameters
     */
    private DownloadEmvParamDto emvParams;
    /**
     * terminal EMV AID parameters
     */
    private List<DownloadEMVDataAid> emvAID;
}

File: lib-model/src/main/java/com/minesec/msa/client/sdk/management/model_k/DownloadParamDto_.kt
package com.minesec.msa.client.sdk.management.model_k

import java.io.Serializable

/**
 * @author eric.song
 * @since 2024/6/19 15:19
 */
data class DownloadParamDto_(
    val emvParams: DownloadEMVParamDto_,
    val emvAID: List<DownloadEMVAidDto_>
): Serializable
